src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -502,6 +502,13 @@ private void ProcessProjectChange(Solution solution, ProjectId projectId)
                 // Nuget may throw an ArgumentException when there is something about the project 
                 // they do not like/support.
             }
+            catch (InvalidOperationException)
+            {
+                // NuGet throws an InvalidOperationException if details
+                // for the project fail to load. We don't need to report
+                // these, and can assume that this will work on a future
+                // project change
```

Comment:
can we be more specific.  Can we only catch this if the exception message text is very specific?  that way if we get this exception for some *other* reason that we still NFW?  This is a pretty broad brush to be catching things with.

Reply:
> can we be more specific 

Unfortunately not, at least not reliably. The string for the exception gets localized, and I don't think we can access that resource here. 

In the current implementation, there's only one instance where it throws this exception. This is from looking through the current NFWs as well as the implementation of this method.

Reply:
I don't usually recommend such things, but exception.StackTrace.Contains("<frame to blame>") might be a reasonable approach, if we do have to go down it.

Reply:
we have NFW for this exception (https://features.vsdata.io/investigatefault?mt=MAC_ADDRESS_HASHES&et=FAULT&sd=2019-01-06&ed=2019-02-05&pn=vs&fn=ide/vbcs/nonfatalwatson&en=nonfatalwatson&fh=13e849c3-da74-b9b7-5ed0-817c663d2226&ex=devenv&ut=external) for last 6 months or so, and all are due to same issue where project is not fully loaded yet. this NFW being there since the beginning of this feature which is at least more than 2 years and the fact that no one looked the NFW shows that this NFW is not much helpful.